common/src/main/java/com/wynntils/services/itemfilter/statproviders/ItemTypeStatProvider.java

```diff
@@ -13,4 +34,32 @@ public class ItemTypeStatProvider extends ItemStatProvider<String> {
     public List<String> getValue(WynnItem wynnItem) {
         return List.of(wynnItem.getClass().getSimpleName().replace("Item", ""));
     }
+
+    @Override
+    public List<String> getValidInputs() {
+        // These are the only WynnItem's that can appear in places where
+        // searching is supported
+        return List.of(
+                AmplifierItem.class.getSimpleName().replace("Item", ""),
+                CharmItem.class.getSimpleName().replace("Item", ""),
+                CraftedConsumableItem.class.getSimpleName().replace("Item", ""),
+                CraftedGearItem.class.getSimpleName().replace("Item", ""),
+                DungeonKeyItem.class.getSimpleName().replace("Item", ""),
+                EmeraldItem.class.getSimpleName().replace("Item", ""),
+                EmeraldPouchItem.class.getSimpleName().replace("Item", ""),
+                GatheringToolItem.class.getSimpleName().replace("Item", ""),
+                GearBoxItem.class.getSimpleName().replace("Item", ""),
+                GearItem.class.getSimpleName().replace("Item", ""),
+                HorseItem.class.getSimpleName().replace("Item", ""),
+                IngredientItem.class.getSimpleName().replace("Item", ""),
+                InsulatorItem.class.getSimpleName().replace("Item", ""),
+                MaterialItem.class.getSimpleName().replace("Item", ""),
+                MultiHealthPotionItem.class.getSimpleName().replace("Item", ""),
+                PotionItem.class.getSimpleName().replace("Item", ""),
+                PowderItem.class.getSimpleName().replace("Item", ""),
+                RuneItem.class.getSimpleName().replace("Item", ""),
+                SimulatorItem.class.getSimpleName().replace("Item", ""),
+                TeleportScrollItem.class.getSimpleName().replace("Item", ""),
+                TomeItem.class.getSimpleName().replace("Item", ""));
```

Comment:
We will forgot to update this. Can we get all inheritors of WynnItem and display all, even if not all will be "real" items?

Reply:
I was going to do that originally but I couldn't find a good way to do it

Reply:
That seems to be tricky, https://stackoverflow.com/questions/205573/at-runtime-find-all-classes-in-a-java-application-that-extend-a-base-class. 

Since we own the code, can't we change the WynnItem supertype to register themselves automatically?

Reply:
I suggest getting `ItemHandler`'s `ItemAnnotator`s, they have the same name as the items, you just replace the "Annotator" part. Otherwise, they should exist whenever an item exists. You can probably Make two subclasses for `ItemAnnotator`, `GameItemAnnotator`, `GuiItemAnnotator` (and leave the last 3 edge cases as normal implementations).